LabExT/Tests/Utils.py

```diff
@@ -26,6 +27,13 @@ def pump_events(self):
             pass
 
 
+def mark_as_laboratory_test(klass):
```

Comment:
```suggestion
def mark_as_laboratory_test(class):
```

Reply:
Changed it to `cls`, since `class` is a reserved keyword.